.circleci/config.yml

```diff
@@ -115,7 +115,7 @@ commands:
             - run:
                 name: Install playwright & ffmpeg
                 command: |
-                  pnpx playwright@1.51.1 install --with-deps
+                  pnpx playwright install --with-deps
```

Comment:
Removing specific version as Renovate couldn't bump it, and it looks like without it, the version in dependencies is used. 👍